docs/getting-started/verify.md

```diff
@@ -32,6 +32,78 @@ The following checks were performed on each of these signatures:
 Getting the above output followed by a JSON array of payloads, ensures you the
 container image was signed using our private key.
 
+## Verify the container SBOMs
+
+We generate SBOMs for all our container image and CLI tools:
+* SBOMs for container images are attached to the corresponding image and can be found in our registry.
+* SBOMs for CLI tools are available as release assets.
```

Comment:
Suggestion to introduce what is SBOM, what's the goal and the scope:

```suggestion
A Software Bill of Materials (SBOM) is a detailed list of all the components of a software. It facilitates security and license compliance assessments. The Inspektor Gadget project publishes SBOMs in the [CycloneDX format](https://cyclonedx.org/specification/overview/) for all our container images and CLI tools:
* SBOMs for container images are attached to the corresponding image and can be found in our [registry](https://github.com/orgs/inspektor-gadget/packages).
* SBOMs for CLI tools are available as [release](https://github.com/inspektor-gadget/inspektor-gadget/releases) assets.

Our image-based gadgets don't contain SBOMs. This could be added later.
```